src/shim/telemetryClient.ts

```diff
@@ -30,13 +30,14 @@ export class TelemetryClient {
     public commonProperties: { [key: string]: string };
     public config: Config;
     private _options: AzureMonitorOpenTelemetryOptions;
+    public configWarnings: string[] = [];
```

Comment:
Update this to be a private method and create a add warning method here so is easy to understand where this is being updated